nengo/config.py

```diff
@@ -84,6 +84,10 @@ def __repr__(self):
         return "<%s[%s]{%s}>" % (type(self).__name__,
                                  self._configures.__name__, ", ".join(params))
 
+    def __setstate__(self, state):
+        for k, v in state.items():
+            setattr(self, k, v)
```

Comment:
I also seem to be able to fix it by checking if `key.startswith('_')` in `__getattr__`, and calling the super if it does (like we do with `__delattr__` and `__setattr__`). Is there any reason not to do this?